storybook-utilities/markdownDocumentationLinkBuilder.js

```diff
@@ -1,6 +1,6 @@
-const markdownDocumentationLinkBuilder = (component) => {
+const markdownDocumentationLinkBuilder = (type, component) => {
```

Comment:
I would make the 'type' default to 'components' to save having to supply 'components' in almost all the story files. We only have 1-3 cases where the type is not going to be components.